packages/pharos/src/components/toggle-button-group/pharos-toggle-button.scss

```diff
@@ -84,6 +79,11 @@
   border-right-width: 0;
 }
 
+:host([variant='secondary']:not([selected])) #button-element:hover pharos-icon {
+  fill: var(--pharos-color-hover-primary);
+  stroke: var(--pharos-color-hover-primary);
+}
+
```

Comment:
This is redundant given what's shown on line 39. See my comment above.

Reply:
this is fixed